@xen-orchestra/web-core/lib/icons/ds-tables-icon.ts

```diff
@@ -0,0 +1,107 @@
+import { defineIconPack } from '@core/packages/icon'
+import {
+  faA,
+  faAlignLeft,
+  faAngleDown,
+  faAngleLeft,
+  faAngleRight,
+  faAnglesLeft,
+  faAnglesRight,
+  faAngleUp,
+  faArrowDown,
+  faArrowDownAZ,
+  faArrowUp,
+  faArrowUpAZ,
+  faCalendar,
+  faCircle,
+  faCirclePlus,
+  faClock,
+  faEye,
+  faEyeSlash,
+  faFilter,
+  faHashtag,
+  faLayerGroup,
+  faSquareCaretDown,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons'
+
+function constructIcon(icon: IconDefinition): any {
+  return {
+    icon,
+    color: 'var(--color-neutral-txt-primary)',
+  }
+}
+
+export const dsTableIcon = defineIconPack({
+  objcet: constructIcon(faA),
+
+  string: constructIcon(faAlignLeft),
+
+  int: constructIcon(faHashtag),
+
+  select: constructIcon(faSquareCaretDown),
+
+  date: constructIcon(faCalendar),
+
+  time: constructIcon(faClock),
+
+  'arraw-up-a-z': constructIcon(faArrowUpAZ),
+
+  'arraw-down-a-z': constructIcon(faArrowDownAZ),
+
+  filter: constructIcon(faFilter),
+
+  'filter-add': [
+    constructIcon(faFilter),
+    {
+      icon: faCircle,
+      color: 'var(--color-neutral-background-primary)',
+      translate: [7, 5.5],
+      size: 13,
+    },
+    {
+      icon: faCirclePlus,
+      color: 'var(--color-neutral-txt-primary)',
+      translate: [7, 5.5],
+      size: 10,
+    },
+  ],
+
+  group: constructIcon(faLayerGroup),
+
+  'group-add': [
+    constructIcon(faLayerGroup),
+    {
+      icon: faCircle,
+      color: 'var(--color-neutral-background-primary)',
+      translate: [7, 5.5],
+      size: 13,
+    },
+    {
+      icon: faCirclePlus,
+      color: 'var(--color-neutral-txt-primary)',
+      translate: [7, 5.5],
+      size: 10,
+    },
+  ],
+
+  show: constructIcon(faEye),
+
+  'show-hide': constructIcon(faEyeSlash),
```

Comment:
```suggestion
  'hide': constructIcon(faEyeSlash),
```
Or
```suggestion
  'hidden': constructIcon(faEyeSlash),
```
I think it's clearer, what do you think ?